apps/server/lib/lexical/server/code_intelligence/completion/builder.ex

```diff
@@ -85,7 +85,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Builder do
     global_boost = Integer.to_string(9 - global_boost)
     local_boost = Integer.to_string(9 - local_boost)
 
-    sort_text = "0#{global_boost}#{local_boost}_#{item.label}"
+    sort_text = "0#{global_boost}#{local_boost}_#{item.sort_text || item.label}"
```

Comment:
heh, super, super nit (and this is my fault)
```suggestion
    sort_text = "0#{global_boost}#{local_boost}_#{fallback(item.sort_text, item.label)}"
```